### ISSUE TITLE
Wrong repo.

### ISSUE THREAD

**Issue by immortal47** [deforum-art/deforum-stable-diffusion#233]
Wrong repo.

_Originally posted by @hithereai in https://github.com/deforum-art/deforum-stable-diffusion/issues/232#issuecomment-1562988537_
            CAN YOU PLEASE SPEAK IN NORMAL LANGUAGE?

**Comment by hithereai** [comment]
Yes, you wanted to post your original issue/ question/ whatever in here:
https://github.com/deforum-art/sd-webui-deforum
